src/modules/dwm.cpp

```diff
@@ -190,6 +191,27 @@ namespace modules {
     m_log.info("%s: Building module", name());
     if (tag == TAG_LABEL_TITLE) {
       m_log.info("%s: Building title", name());
+      if (m_tags_scroll && m_title_label) {
```

Comment:
Are these scroll handlers supposed to be applied to `label-title` or `label-tags`?